client/pluginmanager/csimanager/interface.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	DriverNotFoundErr = errors.New("Driver not found")
+	PluginNotFoundErr = errors.New("Plugin not found")
```

Comment:
This should be `Driver` still - CSI never really calls things plugins

Reply:
resolved over in https://github.com/hashicorp/nomad/pull/7013